googler

```diff
@@ -1175,13 +1206,22 @@ class GoogleParser(html.parser.HTMLParser):
             self.set_handlers_to('result')
             return 'result'
 
+        # Autocorrect
+        if 'spell_orig' in self.classes(attrs) and attrs.get('id') not in {'sifm', 'sifl'}:
+            self.autocorrected = True
+        if tag == 'a' and 'spell' in self.classes(attrs) and attrs.get('id') != 'srfl':
```

Comment:
Can we make add it as an `elif` case to line 1191: `if tag == 'div' and` ...? A tag can't be both `div` and `a` at the same time.